modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/policy/DefaultApplicationPolicyInstance.java

```diff
@@ -66,6 +67,7 @@ private void initPolicyContext() {
     if (policyContext == null) {
       ArtifactContextBuilder artifactBuilder =
           newBuilder().setArtifactType(APP)
+              .setArtifactProperties(new HashMap<>(parametrization.getParameters()))
```

Comment:
the original is immutable, later during the deployment the deployer adds more properties there